src/ServiceInsight/Models/Endpoint.cs

```diff
@@ -2,23 +2,31 @@
 {
     public class Endpoint
     {
-        string host;
-
-        public string Name { get; set; }
+        public string Name
+        {
+            get => EndpointDetails.Name;
+            set => EndpointDetails.Name = value;
+        }
 
         public string Host
         {
-            get { return host ?? HostDisplayName; }
-            set { host = value; }
+            get => EndpointDetails.Host ?? HostDisplayName;
+            set => EndpointDetails.Host = value;
         }
 
-        public string HostId { get; set; }
+        public string HostId
+        {
+            get => EndpointDetails.HostId;
+            set => EndpointDetails.HostId = value;
+        }
 
         public string HostDisplayName { get; set; }
 
-        public string Address => string.Format("{0}{1}", Name, AtMachine());
+        public string Address => $"{Name}{AtMachine()}";
```

Comment:
This does not seem to be related to the task at hand. Suggesting to keep as separate refactoring commit/PR.

Reply:
Discussed with @danielmarbach and we agree that this is minor enough that we can leave as-is this time.